src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -176,7 +176,6 @@ export interface RestorationFilingIF {
     foundingDate: string
   }
   restoration: {
-    date: string // today, as YYYY-MM-DD
```

Comment:
Aren't restorations date-specific? I'm confused why the date has been removed.

Reply:
This "restoration date" field was not needed and was removed from the schema by Vysakh this morning.

This filing still has both a "date" and "effectiveDate", which are set by the API when the filing is filed-and-paid.

If you're wondering about "when does the restoration start", it's when it's filed. If you're wondering about the time period of a limited restoration, that also starts when it's filed -- there is a separate date property, "expiry" for that.